docs/enterprise/active-learning/active_learning.md

```diff
@@ -0,0 +1,202 @@
+# Active Learning
+
+!!! note
+
+    This feature is only available for Roboflow Enterprise users. [Contact our sales team](https://roboflow.com/sales) to learn more about Roboflow Enterprise.
```

Comment:
is that decided?

Reply:
No, this is wrong. We decided to make it available to anyone since it feeds other parts of the app.